src/main/java/de/helmholtz/marketplace/cerebrum/controller/OrganizationController.java

```diff
@@ -0,0 +1,80 @@
+package de.helmholtz.marketplace.cerebrum.controller;
+
+import de.helmholtz.marketplace.cerebrum.entities.Organization;
+import de.helmholtz.marketplace.cerebrum.repository.OrganizationRepository;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestBody;
+import reactor.util.annotation.Nullable;
+
+import java.util.Iterator;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("${spring.data.rest.base-path}/organizations")
+public class OrganizationController {
+
+    private final OrganizationRepository organizationRepository;
+
+    public OrganizationController(OrganizationRepository organizationRepository) {
+        this.organizationRepository = organizationRepository;
+    }
+
+    @GetMapping(
```

Comment:
[non-blocking]: No reason to occupy multiple line, I will recommend that you in-line